steps/commenter/0.0.1/step.yml

```diff
@@ -0,0 +1,67 @@
+title: Commenter
+summary: Create and update comments on GitHub issues and pull requests
+description: Create and update comments on GitHub issues and pull requests
+website: https://github.com/chrisfuller/bitrise-step-commenter
+source_code_url: https://github.com/chrisfuller/bitrise-step-commenter
+support_url: https://github.com/chrisfuller/bitrise-step-commenter/issues
+published_at: 2019-09-08T18:46:11.386798-07:00
+source:
+  git: https://github.com/chrisfuller/bitrise-step-commenter.git
+  commit: 70457ad86bbd0c8a5cb28836c1e4d25f857005fe
+host_os_tags:
+- osx
+- ubuntu
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: chrisfuller/homebrew-tap/commenter
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- commenter_owner: ${BITRISEIO_GIT_REPOSITORY_OWNER}
+  opts:
+    description: |
+      The owner.
+    is_expand: true
+    is_required: true
+    title: --owner
```

Comment:
pls refer this comment: https://github.com/bitrise-io/bitrise-steplib/pull/2124#discussion_r323202409

Reply:
Fixed in `v0.0.2` 👍